app/components/blacklight/hierarchy/facet_field_component.html.erb

```diff
@@ -1,5 +1,6 @@
 <li class="<%= li_class %>" role="treeitem">
-  <%= helpers.facet_toggle_button(field_name, id) if subset.any? %>
+  <% ul_id = SecureRandom.alphanumeric(10) %>
```

Comment:
HTML ids can't start with a number, or weird things can happen.. i don't know if you want to add a static prefix, or...?

Reply:
Oh, good to know. I'll prefix this.